index.js

```diff
@@ -0,0 +1,25 @@
+var express =require("express");
+var hbs =require("express-handlebars");
+var app = express();
+app.set("view engine", "hbs");
+app.get("/", express.static("public"));
+
+app.get("/", function(req, res){
+  res.render("compliments");
```

Comment:
In express, we pass an object to the view, which contains the data for the view.  
```
res.render("compliments", { compliments: a_variable_containing_a_list_of_compliments }
```

The first string that we pass to `res.render` is the name of the view file.  For this call, I expect to see a view file at "views/compliments.hbs".